alpha_integration_dev/old_client.py

```diff
@@ -106,8 +106,11 @@ def log_series(self):
         neptune.log_image('g_img', self.img_path)
 
         # see what we've logged
+        from time import sleep
+        sleep(5)
         logs = neptune.get_experiment().get_logs()
         print(f'Logs: {logs}')
+        assert logs == {'m1': '2.0', 'g_img': None, 'nested/m1': '1.0', 'm2': 'c'}
```

Comment:
* Old API returns strings while new one integer/float values.
* No idea why `g_img` is `None`, but we have the same behavior for old client. `x` and `lastX` values for returned `g_img` channel is equal 1, so images has been uploaded.

Reply:
1. Fixed
2. We do not return images in old client by design. It was always this way.